### PR TITLE
Format the current timestamp into release wiki titles on request

### PR DESCRIPTION
**Pull request by cpennington**

### CODE REVIEW

File: tubular/scripts/update_release_page.py [hunk 1]
```diff
@@ -119,6 +120,8 @@ def create_release_page(
     if title is None:
         title = "{:%Y-%m-%d} Release".format(EXPECTED_RELEASE_DATE)
 
+    title = title.format(timestamp=datetime.now())
```
Comment (jdmulloy) [inline tubular/scripts/update_release_page.py:128]: Does this work? What is the content of title when before we get to this line?

**Review by jdmulloy — CHANGES_REQUESTED** [review]

File: tubular/scripts/update_release_page.py (hunk 1 above)
Comment (cpennington) [reply tubular/scripts/update_release_page.py:128]: This basically allows callers to use a title like `"{timestamp:%Y-%m-%d} Release"` to get a page like we have now, or to add more detail (like I do in the linked PR). If the title doesn't have a format field in it, then nothing is changed.

Comment (jdmulloy) [reply tubular/scripts/update_release_page.py:128]: Got it.

**Review by jdmulloy — APPROVED** [review]

File: tubular/scripts/update_release_page.py (hunk 1 above)
Comment (jdmulloy) [reply tubular/scripts/update_release_page.py:128]: @cpennington Can you add a comment above this explaining this?

**Review by jdmulloy — CHANGES_REQUESTED** [review]

**Review by jdmulloy — APPROVED** [review]

**Pull request merged**